src/navigator/Navigator.tsx

```diff
@@ -685,6 +686,10 @@ function nativeBottomSheets(BottomSheet: typeof RootStack) {
         name={Screens.FiatExchangeCurrencyBottomSheet}
         component={FiatExchangeCurrencyBottomSheet}
       />
+      <BottomSheet.Screen
```

Comment:
curious why you chose a screen over a bottom sheet component directly inside the transaction details component? it seems like the only place that needs to launch this bottom sheet is the details screen (the components we have registered as screens so far are ones that can launched from many places...)